docs/website/layouts/partials/docs/navbar-link.html

```diff
@@ -0,0 +1,19 @@
+{{/* Expect to have context bound to ".ctx", and the top level
+     "version" and "pageUrl" passed in with the context.
+ */}}
+ {{- $latest     := index site.Data.releases 0 -}}
```

Comment:
FYI: the dashes aren’t necessary in templates, as the HTML output is minified. Not a blocker, but could save you quite a few keystrokes in the future.